t/rose-metadata-check/lib/custom_widget.py

```diff
@@ -47,8 +47,8 @@ def __init__(self, value, metadata, set_value, hook, arg_str=None):
             tooltip_text = self.WARNING_MESSAGE.format(value,
                                                        int_value)
         my_adj = gtk.Adjustment(value=int_value,
```

Comment:
Does this file work? It uses GTK?

Reply:
One might think that it wouldn't. That suggests that this might be a hole in test coverage... :cry: